distributed/distributed-schema.yaml

```diff
@@ -339,6 +339,17 @@ properties:
               To avoid this, Dask usually used a file-based lock.
               However, on some systems file-based locks don't work.
               This is particularly common on HPC NFS systems, where users may want to set this to false.
+          transfer:
```

Comment:
XREF: #6977 

I went with singular `transfer` to be coherent with variable naming.